4.3/README.md

```diff
@@ -42,9 +43,17 @@ If you want to check that your data import was successful, you can use the API w
 
 The following environment variables are available for configuration:
 
-- `PBF_URL`: Which [OSM extract](#openstreetmap-data-extracts) to download and import. It cannot be used together with `PBF_PATH`. Check [https://download.geofabrik.de](https://download.geofabrik.de)
+- `PBF_URL`: Which [OSM extract](#openstreetmap-data-extracts) to download and import. It cannot be used together with `PBF_PATH`.
+  Check [https://download.geofabrik.de](https://download.geofabrik.de) 
+  Since de DL speed is restrictet at Geofabrik, for importing the full planet there is a recommended list of mirrors at the [OSM Wiki](https://wiki.openstreetmap.org/wiki/Planet.osm#Planet.osm_mirrors).
```

Comment:
```suggestion
  Since the download speed is restricted at Geofabrik, there is a recommended list of mirrors for importing the full planet at [OSM Wiki](https://wiki.openstreetmap.org/wiki/Planet.osm#Planet.osm_mirrors).
```

Reply:
You can click "Commit suggestion" to add this edit to your branch.